VK/VK/FriendListScreen/FriendListTableViewController.swift

```diff
@@ -53,114 +46,83 @@ final class FriendListTableViewController: UITableViewController, UIViewControll
     // MARK: - UITableViewDataSource
 
     override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
-        guard let char = lettersCountMap[safe: section]?.key else { return nil }
-        return String(char)
+        let sortedKeys = sortedFriendsMap.keys.sorted()
+        return String(sortedKeys[section])
     }
 
     override func numberOfSections(in tableView: UITableView) -> Int {
-        sortedlettersMap.count
+        sortedFriendsMap.keys.count
     }
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        guard let number = lettersCountMap[safe: section]?.value else { return 1 }
-        return number
+        let sortedKeys = sortedFriendsMap.keys.sorted()
+        let key = sortedKeys[section]
+        return sortedFriendsMap[key]?.count ?? Constants.defaultRawValue
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard
             let cell = tableView.dequeueReusableCell(
                 withIdentifier: CellIdentifiers.commonGroupTableViewCellID,
                 for: indexPath
-            ) as? CommonGroupTableViewCell else { return UITableViewCell() }
+            ) as? CommonGroupTableViewCell
+        else { return UITableViewCell() }
 
-        guard usersAtSections.count > 1 else { return UITableViewCell() }
-        cell.configureCell(unit: usersAtSections[indexPath.section][indexPath.row], labelNameTapHandler: tapHandler)
+        let sortedKeys = sortedFriendsMap.keys.sorted()
+        let key = sortedKeys[indexPath.section]
+        guard
+            let friendsListSection = sortedFriendsMap[key],
+            let friendsInfo = friendsListSection[safe: indexPath.row]
+        else { return UITableViewCell() }
+        cell.configureCell(unit: friendsInfo, labelNameTapHandler: tapHandler)
         return cell
     }
 
     // MARK: - Private methods
 
-    private func clearAllArrays() {
-        users.removeAll()
-        sortedUsers.removeAll()
-        lettersCountMap.removeAll()
-        sortedlettersMap.removeAll()
-        usersAtSections = [[]]
-        lettersMap = Alphabets.russianLettersMap
+    private func configureScreen() {
+        fetchFriends()
+        setupScene()
+    }
+
+    private func fetchFriends() {
+        NetworkService.shared.fetchFriends { friends in
```

Comment:
```suggestion
        NetworkService.fetchFriends { friends in
```
shared вот так вот не надо)